src/fides/api/ops/api/v1/endpoints/privacy_experience_endpoints.py

```diff
@@ -81,56 +70,34 @@ def privacy_experience_list(
             PrivacyExperience.disabled.is_(False)
         )
     if region is not None:
-        experience_query = experience_query.filter(
-            PrivacyExperience.regions.contains([region])
-        )
+        experience_query = experience_query.filter(PrivacyExperience.region == region)
     if component is not None:
         experience_query = experience_query.filter(
             PrivacyExperience.component == component
         )
+    if has_config is True:
+        experience_query = experience_query.filter(
+            PrivacyExperience.experience_config_id.isnot(None)
+        )
+    if has_config is False:
+        experience_query = experience_query.filter(
+            PrivacyExperience.experience_config_id.is_(None)
+        )
 
     results: List[PrivacyExperience] = []
     for privacy_experience in experience_query.order_by(
         PrivacyExperience.created_at.desc()
     ):
         privacy_notices: List[
             PrivacyNotice
-        ] = privacy_experience.get_related_privacy_notices(db, region, show_disabled)
+        ] = privacy_experience.get_related_privacy_notices(db, show_disabled)
```

Comment:
A PrivacyExperience's notices aren't filtered on region anymore, since a PrivacyExperience is only associated with one region.  We just filter then any notices that have the same region as the experience, plus need to be displayed in the same way.